nginx/nginx.conf

```diff
@@ -63,6 +67,7 @@ http {
     location @app {
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header Host $http_host;
+	  proxy_set_header X-Forwarded-Proto https;
```

Comment:
Please fix spacing so they line up.

Reply:
I'm not sure why it's showing like this. Opening the file in GitHub and notepad shows it indented normally for me.